src/Rewrite/ErrorProcessingAggregatorLogging.php

```diff
@@ -46,6 +46,7 @@ public function getRowsGroupedByErrorCode(
                 $this->logger->info('export/import error, ErrorMessage:', [$error->getErrorMessage()]);
                 /** @todo temporary output rownumber, remove var_dump when logging is finished */
                 var_dump($error->getRowNumber());
+                var_dump($error->getErrorMessage());
```

Comment:
Debug `var_dump`s?

Reply:
Ik krijg de logging directe regel erboven niet aan de praat.

Reply:
Heb ik tijdelijk de var_dump toegevoegd.

Reply:
Waarom staat de __construct eigenlijk onderaan in deze file?

Reply:
Bij deze bovenaan geplaatst, had geen specifieke reden.